daVinci:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  setup1_position_controller:
    type: position_controllers/JointPositionController
    joint: setup1_joint
    pid: {p: 10.0, i: 1.0, d: 0.5}

  setup2_position_controller:
    type: position_controllers/JointPositionController
    joint: setup2_joint
    pid: {p: 10.0, i: 1.0, d: 0.5}

  setup3_position_controller:
    type: position_controllers/JointPositionController
    joint: setup3_joint
    pid: {p: 10.0, i: 1.0, d: 0.5}

  outer_yaw_position_controller:
    type: position_controllers/JointPositionController
    joint: outer_yaw_joint
    pid: {p: 10.0, i: 1.0, d: 0.5}

  outer_pitch_position_controller:
    type: position_controllers/JointPositionController
    joint: outer_pitch_back_joint
    pid: {p: 10.0, i: 1.0, d: 0.5}